less/admin/DashboardPage.less

```diff
@@ -1,18 +1,18 @@
 .DashboardPage {
-  background: @control-bg;
+  background: @body-bg;
   color: @control-color;
   min-height: 100vh;
 
   @media @desktop-up {
-    .container {
+    > .container {
       padding: 30px;
```

Comment:
This padding is now inconsistent with that of other pages

Reply:
I'm experiencing the opposite, with the arrow it is consistent, without, it is extra big.

Reply:
It's not the arrow as much as the 30px: that 30px is unnecessarily applied to the top.

Reply:
Right, I see now